exercises/hangman.py

```diff
@@ -0,0 +1,62 @@
+# Variablen
+gesucht = 'test'
+
+gefunden = []
+falsch_geraten = []
+
+
+def show():
+    print('Falsche Buchstaben:', falsch_geraten)
+    for buchstabe in gesucht:
+        if buchstabe in gefunden:
+            print(buchstabe, end=' ')  # schreibt die erratenen Buchstaben
+        else:
+            # schreibt die Lücken der noch zu erratenen Buchstaben
+            print('_', end=' ')
+    print('')
+
+
+def is_valid(inp):  # inp ist der Input
+    return True
+
+
+def eingabe():
+    buchstabe = input('Buchstabe? ')
+    # Buchstabe wird in is_valid eingefügt/ falls is_valid false ausgibt
```

Comment:
Hmm, ich interpretiere Ihren Kommentar, als würde der Buchstabe nur dann in `is_valid` eingefügt, falls `is_valid` False ausgibt?
```suggestion
    # Solange der eingegebene Buchstabe nicht valid ist (daher is_valid False zurückgibt), wird ein neuer Buchstabe abgefragt
```